src/utils/displayFunctions.ts

```diff
@@ -111,7 +111,10 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
   };
 
   const displayPriceTimestamp = (price: PriceDescription) => {
-    return new Date(Number(price.timestamp) * 1000).toUTCString();
+    return new Intl.DateTimeFormat(navigator.language, {
+      timeStyle: 'medium',
+      dateStyle: 'short',
+    }).format(new Date(Number(price.timestamp) * 1000));
```

Comment:
glad to see the formatting. what timezone is `timestamp` in? what assumptions about timezone does `new Date` make?

Reply:
From my understanding timestamp we get from the chain is epoch time, so no timezone offset, but denoted in seconds rather than milliseconds. And the `Date` constructor expects epoch time in milliseconds.